### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by Jason-Cooke**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Highlights for developers
   * extensible with plugin system
 * interfaced backends for documents parsing and rendering
   * high quality document backend libraries like MuPDF, DjvuLibre and CREngine
-  * interacting with frontend via LuaJIT FFI for best performence
+  * interacting with frontend via LuaJIT FFI for best performance
```
Comment (Frenzie) [inline README.md]: Besides the typo I'd also rephrase it slightly. 
```suggestion
  * frontend interaction via LuaJIT FFI for best performance
```

Comment (Frenzie) [reply README.md]: @Jason-Cooke Could you please commit (or object to/comment on) my suggestion? :-)

Comment (Jason-Cooke) [reply README.md]: @Frenzie Ya sure

**Comment by Frenzie** [comment]
Thanks!

**Comment by Jason-Cooke** [comment]
You're very welcome :)

**Pull request merged**